src/odemis/acq/fastem.py

```diff
@@ -439,6 +450,8 @@ def pre_calibrate(self):
         if not fastem_calibrations:
             raise ModuleNotFoundError("Need fastem_calibrations repository to run pre-calibrations.")
 
+        asm_config_orig = None
+
```

Comment:
What's the point of initializing it to `None`? Is it so that even if get_config_asm() fails, the `finally` block doesn't fail? However, configure_asm() needs a dict properly field up, so passing None will fail anyway...

Reply:
Correct. I have an extension for the `configure_asm()` case ready. I will push a new commit for this here.

Reply:
Sorry, I will make a pr in the fastem-calibration repo for this not here.